src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/FilterColumnsWithAllNullValues.java

```diff
@@ -0,0 +1,37 @@
+package com.northconcepts.datapipeline.foundations.examples.pipeline;
+
+import com.northconcepts.datapipeline.foundations.file.LocalFile;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Column;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Dataset;
+import com.northconcepts.datapipeline.foundations.pipeline.input.ExcelPipelineInput;
+
+public class FilterColumnsWithAllNullValues {
+
+    public static void main(String[] args) {
+
+        Pipeline pipeline = new Pipeline();
+
+        LocalFile inputFile = new LocalFile()
+                .setName("Input File")
+                .setPath("PATH_TO_INPUT_FILE.csv")
+                .detectFileType();
+
+        ExcelPipelineInput pipelineInput = new ExcelPipelineInput()
+                .setFileSource(inputFile)
+                .setFieldNamesInFirstRow(true);
+
+        pipeline.setInput(pipelineInput);
+
+        Dataset dataset = pipeline.getDataset();
+        dataset.load().waitForRecordsToLoad();
+
+        for(Column column : dataset.getColumns()) {
```

Comment:
We need to filter the resulting data set.  Let's add a command (or multiple commands) to remove the fully-null columns and write the output to a new file in https://github.com/NorthConcepts/DataPipeline-Examples/tree/master/data/output.

Reply:
done